android/src/main/java/com/swmansion/rnscreens/ScreenViewManager.kt

```diff
@@ -144,6 +144,8 @@ class ScreenViewManager : ViewGroupManager<Screen>(), RNSScreenManagerInterface<
 
     override fun setTransitionDuration(view: Screen?, value: Int) = Unit
 
+    override fun setHideKeyboardOnSwipe(view: Screen?, value: Boolean) = Unit
```

Comment:
What about adding new props to interface and delegate so it works on old arch?

Reply:
As I'm testing these things on [last PR](https://github.com/software-mansion/react-native-screens/pull/1446) I'll copy the interfaces there, so I don't have to do it for every prop.